koans/TriangleProject2Koans.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace PhpKoans;
+require_once 'Triangle.php';
+use PHPUnit\Framework\TestCase;
+
+# You need to finish implementing triangle() in the file 'triangle.php'
+class TriangleProject2Koans extends TestCase
+{
+
+    private Triangle $triangle;
+
+    /**
+     * @setUp
+     */
+    protected function setUp() : void
+    {
+        $this->triangle = new Triangle();
```

Comment:
You forgot to put `parent::setUp();` at the beggining of the setUp().
If we don't put this, the tests will fail.